### PR TITLE
Market Picker: Tracking

### PR DESCRIPTION
**Pull request by hedvigoscar**
- [X] Functionality is covered by an integration test
- [ ] Functionality is accessible in engineering mode

### CODE REVIEW

File: app/src/androidTest/java/com/hedvig/app/feature/marketpicker/KnownGeoTest.kt [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+            await atMost TWO_SECONDS untilAsserted {
+                save {
+                    click()
+                }
+            }
+        }
+
+        verify(exactly = 0) { tracker.selectMarket(any()) }
+        verify(exactly = 1) { tracker.selectLocale(Language.SV_SE) }
+        verify(exactly = 1) { tracker.submit() }
```
Comment (MathiasOhrstrom) [inline app/src/androidTest/java/com/hedvig/app/feature/marketpicker/KnownGeoTest.kt:109]: This is cool 😎

File: app/src/main/java/com/hedvig/app/feature/marketpicker/MarketPickerActivity.kt [hunk 2]
```diff
@@ -97,3 +102,23 @@ class MarketPickerActivity : BaseActivity(R.layout.activity_market_picker) {
         fun newInstance(context: Context) = Intent(context, MarketPickerActivity::class.java)
     }
 }
+
+class MarketPickerTracker(
+    private val mixpanel: MixpanelAPI
+) {
```
Comment (MathiasOhrstrom) [inline app/src/main/java/com/hedvig/app/feature/marketpicker/MarketPickerActivity.kt:108]: Could be in its own file

File: app/src/androidTest/AndroidManifest.xml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="com.hedvig.app">
+
+    <application android:extractNativeLibs="true" />
```
Comment (MathiasOhrstrom) [inline app/src/androidTest/AndroidManifest.xml:5]: Did this help decrease the .APK size?

**Review by MathiasOhrstrom — CHANGES_REQUESTED** [review]

File: app/src/androidTest/AndroidManifest.xml (hunk 3 above)
Comment (hedvigoscar) [reply app/src/androidTest/AndroidManifest.xml:5]: No - it is however required to use `MockK`. It only applies for builds with the `androidTest`-configuration, so it should not affect production builds at all.

File: app/src/androidTest/java/com/hedvig/app/feature/marketpicker/KnownGeoTest.kt (hunk 1 above)
Comment (hedvigoscar) [reply app/src/androidTest/java/com/hedvig/app/feature/marketpicker/KnownGeoTest.kt:109]: It's pretty nifty, yeah! 😎

**Pull request merged**